### PR TITLE
Move English translation to separate directory and fix blog urls

### PR DESCRIPTION
**Pull request by stcherenkov**

### CODE REVIEW

**Review by Dragonesis — APPROVED** [review]

File: .vuepress/src/components/base/MainHeader/SelectLanguage/SelectLanguage.vue [hunk 1]
```diff
@@ -50,23 +49,27 @@ export default {
...
-            return { text, link }
+            return { text, link, lang }
           })
         }
       }
     },
   },
+  methods: {
+    handleClick(lang) {
+      console.log(lang);
```
Comment (stcherenkov) [inline .vuepress/src/components/base/MainHeader/SelectLanguage/SelectLanguage.vue]: console.log

File: .vuepress/theme/layouts/Layout.vue [hunk 2]
```diff
@@ -129,22 +132,23 @@ export default {
...
     })
   },
   mounted() {
+    this.$cookies.set('lang', this.$localeConfig.shortLang, '360d')
     Vue.use(VueGtm, {
       id: process.env.GOOGLE_TAG_MANAGER_ID,
       enabled: true,
       debug: process.env.NODE_ENV === 'development',
-      vueRouter: this.$router,
+      // vueRouter: this.$router,
```
Comment (stcherenkov) [inline .vuepress/theme/layouts/Layout.vue]: Зачем он закомментирован тут? Может быть эту строку надо удалить, если она не используется?

File: nginx.tpl.conf [hunk 3]
```diff
@@ -33,17 +33,37 @@ http {
...
+
     server {
         listen 80 default_server;
 
         error_page   404  /404.html;
 
         rewrite ^/assets/white_paper.* https://github.com/jibrelnetwork/white_paper permanent;
 
+        location = / {
+            return 302 $scheme://$http_host/$lang/;
```
Comment (soar) [inline nginx.tpl.conf]:
```suggestion
            return 302 $scheme://$host/$lang/;
```
In the downstream container, you will always have a port number in this variable. You shouldn't use it in redirects.

File: nginx.tpl.conf [hunk 4]
```diff
@@ -33,17 +33,37 @@ http {
...
+        }
+
+        location ~* "^/(about|blog|careers|enterprise|news)" {
+            return 301 $scheme://$http_host/$lang$request_uri;
```
Comment (soar) [inline nginx.tpl.conf]:
```suggestion
            return 301 $scheme://$host/$lang$request_uri;
```
Same problem

**Review by soar — CHANGES_REQUESTED** [review]

File: nginx.tpl.conf [hunk 5]
```diff
@@ -33,17 +33,37 @@ http {
...
     gzip_types *;
 
+    map $cookie_lang $lang {
+        default 'en';
+        zh      zh;
+        ko      ko;
+    }
+
     server {
         listen 80 default_server;
```
Comment (soar) [inline nginx.tpl.conf:43]:
```suggestion
        listen 80 default_server;
        
        port_in_redirect off;
```
See https://github.com/jibrelnetwork/coinmena-web/pull/131

File: nginx.tpl.conf [hunk 6]
```diff
@@ -33,17 +33,39 @@ http {
...
+        error_page   404  /$lang/404.html;
 
         rewrite ^/assets/white_paper.* https://github.com/jibrelnetwork/white_paper permanent;
 
+        location = / {
+            return 302 $scheme://$host/$lang/;
+        }
+
+        location ~* "^/(about|blog|careers|enterprise|news)" {
+            return 301 $scheme://$host/$lang$request_uri;
```
Comment (soar) [inline nginx.tpl.conf]: Maybe it's a bad idea to use permanent redirect with a variable?

Comment (stcherenkov) [reply nginx.tpl.conf]: Fixed!

**Pull request merged**